doc/kernel/memory_management/demand_paging.rst

```diff
@@ -133,12 +140,24 @@ called from the kernel paging code:
   The function returns a pointer to the page frame corresponding to
   the selected data page.
 
-Currently, a NRU (Not-Recently-Used) eviction algorithm has been
-implemented as a sample. This is a very simple algorithm which
-ranks each data page on whether they have been accessed and modified.
-The selection is based on this ranking.
+There is one additional function which is called by the architecture's memory
+management code to flag data pages when they trigger an access fault:
+:c:func:`k_mem_paging_eviction_accessed()`. This is used by the LRU algorithm
+to requeue "used" pages.
+
+Two eviction algorithms are currently available:
+
+* A NRU (Not-Recently-Used) eviction algorithm has been implemented as a
+  sample. This is a very simple algorithm which ranks each data page on
+  whether they have been accessed and modified. The selection is based on
```

Comment:
Either ...
   they have -> it has
or ...
   each data page -> data pages

(but not both).